### DISCUSSION TITLE
Integrate Airbyte

### DISCUSSION THREAD

**Discussion by benjamin-steurer** [coollabsio/coolify#2421] [Service Template Requests]
It would be great to integrate Airbyte into coolify

**Comment by Shadowfita** [comment]
+1 !!

*edit*

I just found out that they've discontinued docker support. 😐

**Reply by feliche93** [reply]
Would also be super interested in this 🤔

**Comment by wiselancer** [comment]
+1

**Comment by simplerethink** [comment]
I would like support for this too.

Website: https://airbyte.com/
Github: https://github.com/airbytehq/airbyte
Docker: https://docs.airbyte.com/platform/using-airbyte/getting-started/oss-quickstart

**Comment by tbsvttr** [comment]
Would be very cool.

**Comment by romainds-tech** [comment]
Would be very cool and needed

**Comment by oguzydz** [comment]
any update on this?

**Comment by aleksandarbasara** [comment]
Any update on this?

**Comment by LFDEVBR** [comment]
Any update on this?

**Comment by nktnet1** [comment]
I've successfully constructed a standalone docker compose template for Airbyte here:

- https://github.com/nktnet1/airbyte-compose

Coolify template:

- https://github.com/coollabsio/coolify/pull/11568

---

As an aside, I've also made a corresponding template for Dokploy:

- https://github.com/Dokploy/templates/pull/1125

**Reply by nktnet1** [reply]
Would appreciate if anyone can help test out the template in #11568 :).

If you need a source DB, you can use:

- https://rnacentral.org/help/public-database

CC
@benjamin-steurer
@feliche93
@Shadowfita
@wiselancer
@simplerethink
@tbsvttr
@romainds-tech
@oguzydz
@aleksandarbasara 
@LFDEVBR
@mariusvilholm (from https://github.com/coollabsio/coolify/issues/345)

in case you are still interested in this.

**Reply by aleksandarbasara** [reply]
Will do this week
